src/main/scala/tech/sourced/gemini/Gemini.scala

```diff
@@ -211,13 +194,14 @@ class Gemini(session: SparkSession, log: Slf4jLogger, keyspace: String = Gemini.
     }
   }
 
-  def extractFeatures(client: FeatureExtractor, uast: Node): Iterable[Feature] = {
-    val idRequest = IdentifiersRequest(uast=Some(uast), docfreqThreshold=5)
-    val litRequest = LiteralsRequest(uast=Some(uast), docfreqThreshold=5)
-    val graphletRequest = GraphletRequest(uast=Some(uast), docfreqThreshold=5)
+  def extractFeatures(client: FeatureExtractor, uast: Option[Node]): Iterable[Feature] = {
```

Comment:
no need to call FE when `uast=None`

Reply:
Question: why is there a `client.identifiers(idRequest)` at https://github.com/src-d/gemini/pull/101/files#diff-747eed6d907b7380e439aec919e624c6L218, before reading results in a for loop after?

Reply:
wow! I didn't see it. Leftover most probably. Please remove. Thank you!